docs/source/data/subscriptions.mdx

```diff
@@ -5,6 +5,8 @@ description: Persistent GraphQL read operations
 
 import {ExpansionPanel} from 'gatsby-theme-apollo-docs';
 
+FIXME AS3: Whole file needs to be rewritten, as we don't have built in subscription support any more, and probably shouldn't recommend using `subscriptions-transport-ws` at this time.
```

Comment:
I will gladly help overhaul, but I definitely don't know the "right" way to get this functionality working in 3

Reply:
@trevor-scheer might also be interested in this.

Reply:
(At some point we should PR to `graphql-ws` updating their README.)

Reply:
Note to self: there's a github backed codesandbox to update here